internal/pkg/download/manager.go

```diff
@@ -1,6 +0,0 @@
-package download
-
-type DownloadManager interface {
```

Comment:
The original struct and interfaces are removed because it's not needed given the logic is simple.